fe1-web/components/__tests__/UserListItem.test.tsx

```diff
@@ -18,23 +31,38 @@ describe('UserListItem', () => {
   it('calls subscribeToChannel correctly when clicking on follow', () => {
     const expectedChannel = '/root/LaoId/social/PublicKey';
     const button = render(
-      <UserListItem laoId={laoId} publicKey={publicKey} />,
+      <UserListItem laoId={laoId} publicKey={publicKey} currentUserPublicKey={mockPublicKey} />,
     ).getByText(STRINGS.follow_button);
     fireEvent.press(button);
     expect(subscribeToChannel).toHaveBeenCalledTimes(1);
     expect(subscribeToChannel).toHaveBeenCalledWith(expectedChannel);
   });
 
+  it('calls navigate correctly when clicking on profile', () => {
+    const { getByText } = render(
+      <UserListItem laoId={laoId} publicKey={publicKey} currentUserPublicKey={mockPublicKey} />,
+    );
+    const followButton = getByText(STRINGS.follow_button);
+    fireEvent.press(followButton);
+    const profileButton = getByText(STRINGS.profile_button);
+    fireEvent.press(profileButton);
+    expect(mockNavigate).toHaveBeenCalledTimes(1);
```

Comment:
is it necessary to have this?

Reply:
Yes it is, since the button to access to someone's profile is only clickable when you follow them. Then, for the test, I had to click on follow otherwise the button is unclickable.